django_saml2_auth_lw/urls.py

```diff
@@ -1,10 +1,10 @@
-from django.conf.urls import url
+from django.urls import path
 from . import views
 
 app_name = 'django_saml2_auth'
 
 urlpatterns = [
-    url(r'^acs/$', views.acs, name="acs"),
-    url(r'^welcome/$', views.welcome, name="welcome"),
-    url(r'^denied/$', views.denied, name="denied"),
+    path('acs', views.acs, name="acs"),
+    path('welcome', views.welcome, name="welcome"),
+    path('denied', views.denied, name="denied"),
 ]
```

Comment:
okay, yep this looks correct
https://stackoverflow.com/questions/47947673/is-it-better-to-use-path-or-url-in-urls-py-for-django-2-0

Reply:
it looks like url is just an alias of `path` anyways https://docs.djangoproject.com/en/3.0/ref/urls/#url as per this

Reply:
I think you should not remove the regex pattern of this one, so probably use `re_path` as per written in https://docs.djangoproject.com/en/3.0/ref/urls/#url

Reply:
all the regex start with `^` (beginning of line) and end with `$` (end of line) and don't really use any fancy regex features, so we can just use `path` without needing to use regex

Reply:
ahh okay sound good